angular.js

```diff
@@ -32,5 +32,28 @@ module.exports = {
             Overrides to Angular extra recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended--extra.json
         */
+
+        /*
+            Overrides to Angular rules outside of the recommended configuration:
+        */
+
+        '@angular-eslint/component-max-inline-declarations': 'error',
```

Comment:
[@angular-eslint/component-max-inline-declarations](http://codelyzer.com/rules/component-max-inline-declarations)

Reply:
Worthy of consideration, but I would increase the defaults.

**Proposal**
```javascript
'component-max-inline-declarations': [true, {"animations": 15, "styles": 15, "template": 15}]
```

Reply:
Any particular rationale for increasing the defaults? I like the consistency of being able to find styles and templates in their own files and would only use inline declarations for extremely trivial cases (1 or 2 lines, which aligns well with the default of `3` for `styles` and `template`).

**Proposal**: `        '@angular-eslint/component-max-inline-declarations': 'error',`

Reply:
Yes, I do inline by default in StackBlitz, and so I've gotten a lot of time with it. You can't do anything with three lines of CSS. Three lines of HTML is annoying when you need four, five, or six is typical for three lines per element. The class probably won't be much either, so the rationale is that ~45 lines fits nicely when you should have already considered separate files anyway. If you want to bring it down significantly, I would say 15, 15, and 6 for two elements or attributes with line length.

Reply:
@TrevorKarjanis this is ready to close